tests/diff_tests.rs

```diff
@@ -119,46 +116,6 @@ fn diff_meta(ours: &Hierarchy, ref_header: &vcd::Header) {
     }
 }
 
-fn waveform_scope_type_to_string(tpe: ScopeType) -> &'static str {
```

Comment:
this functionality was meant as a quick implementation to get the diff test working. This should not be part of the `wellen` API.